testsuite/features/support/lavanda.rb

```diff
@@ -44,21 +44,11 @@ def full_hostname
     @in_full_hostname
   end
 
-  def private_ip
-    raise 'empty private_ip, something wrong' if @in_private_ip.empty?
-    @in_private_ip
-  end
-
```

Comment:
Keep it please. Let's only remove unused step definitions, but let's keep what we offer as part of our modules.